src/OSPSuite.Core/Domain/Services/SimModelBatch.cs

```diff
@@ -49,8 +49,7 @@ public void InitializeForSensitivity()
 
       private Simulation createAndFinalizeSimulation(IReadOnlyList<string> variableParameterPaths, IReadOnlyList<string> variableMoleculePaths)
       {
-         var simulationExport = CreateSimulationExport(_modelCoreSimulation, SimModelExportMode.Optimized, 
-            TreatConstantMoleculesAsParameters ? SimulationExportCreatorMode.TreatConstantMoleculeAsParameter : SimulationExportCreatorMode.ConstantMoleculeStaysMolecule);
+         var simulationExport = CreateSimulationExport(_modelCoreSimulation, SimModelExportMode.Optimized, variableMoleculePaths);
```

Comment:
if we add another parameter, we should think about creating an option Parameter for this. That way you do not have to modify all callers but instead you get always the same signature. just FYI